commitizen/commands/bump.py

```diff
@@ -190,6 +192,9 @@ def __call__(self) -> None:  # noqa: C901
                     "--prerelease-offset cannot be combined with MANUAL_VERSION"
                 )
 
+            if get_next:
```

Comment:
Probably not required in this PR, but I feel we could move these validations out as a separate function 🤔